src/components/Header/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './Header.module.scss';
```

Comment:
Be consistent in the order of imports and try to group related things together. External libraries, components, styles, etc.